src/Surfnet/StepupGateway/GatewayBundle/Entity/SecondFactor.php

```diff
@@ -103,4 +103,12 @@ public function getLoaLevel()
     {
         return (new SecondFactorType($this->secondFactorType))->getLevel();
     }
+
+    /**
+     * @return bool
+     */
+    public function requiresGssf()
```

Comment:
I think a second factor *is* a Generic Saml Second Factor, and does not *require* it.